src/benchmarks/real-world/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
-  <Import Project="$(MSBuildThisFileDirectory)\..\..\..\common.props" />
```

Comment:
Don't you need to import the new `Directory.Build.props` here, which now imports `Arcade SDK`?

Reply:
No, Direcotry.Build.props/targets imports are implicit for Microsoft.NET.Sdk projects.